packages/mgt-components/src/components/mgt-person-card/mgt-person-card.scss

```diff
@@ -123,10 +138,60 @@ $person-card-details-item-font-size: var(--person-card-details-item-font-size, 1
   }
 
   .section-nav {
-    display: flex;
-    padding: 0 14px;
     border-bottom: 1px solid set-var(tab__line__color, $theme-default, $common);
+    height: 36px;
+    padding-bottom: 7px;
+    fluent-tabs {
+      background-color: $background-color;
+      grid-template-columns: minmax(1px, 0.1fr);
+      .activeIndicator {
+        width: 40px !important;
+      }
+      .tablist {
+        padding-left: 20px !important;
+      }
+      fluent-tab-panel {
+        max-height: 360px;
+        min-height: 360px;
+        overflow-y: auto;
+        overflow-x: hidden;
+        padding: 0px;
+        border-top: 1px solid set-var(tab__line__color, $theme-default, $common);
+        .inserted {
+          max-width: var(--mgt-flyout-set-width, 375px);
+          min-width: var(--mgt-flyout-set-width, 360px);
+        }
+        .overviewPanel {
+          max-width: var(--mgt-flyout-set-width, 375px);
+        }
 
+        &::-webkit-scrollbar {
+          height: 4em;
+          width: 4px;
+          border-radius: 11px;
+        }
+        &::-webkit-scrollbar-button {
+          background: $background-color;
+          height: 1px;
+        }
+        &::-webkit-scrollbar-track {
+          background: $background-color;
+          border-radius: 10px;
+        }
+        &::-webkit-scrollbar-track-piece {
+          background: $background-color;
+        }
+        &::-webkit-scrollbar-thumb {
+          background: grey;
+          border-radius: 10px;
+          height: 4px;
+        }
+      }
+    }
+    fluent-tab {
+      padding-bottom: 4px !important;
+      margin-bottom: -3px !important;
+    }
```

Comment:
These can be styled as part of `.section-nav__icon` and remove !important because `.section-nav__icon` took selection privilege. 
```css
.section-nav__icon {
  margin-bottom: -3px;
  padding: 12px 12px 4px;
}
```

Reply:
Found the tab slots had margin modifiers we were able to use. No longer relevant.